specification.md

```diff
@@ -0,0 +1,1199 @@
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+#### Response Metadata
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`format`</td>
+    <td>Recommended.</td>
+    <td>The mime type of the video.  If not provided, the format may be guessed from the extension or the Content-Type header of the resource.</td>
+    <td>"video/mp4"</td>
+  </tr>
+  <tr>
+    <td>`language`</td>
+    <td>Optional</td>
+    <td>The ISO 639-3 code for the language of the response, if known.</td>
+    <td>{'language':'eng'}</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### audio
+
+Represents an audio recording submitted by the Contact
+
+#### Response format
+
+A string with the URL where the audio can be retrieved. (TODO: Do we want to support inline audio data?)
+
+```
+"https://myexampleflowserver.com/resources/audio/23429837435.png"
+```
+
+#### Type options (type_options)
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+#### Response Metadata
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`format`</td>
+    <td>Recommended.</td>
+    <td>The mime type of the audio.  If not provided, the format may be guessed from the extension or the Content-Type header of the resource.</td>
+    <td>"audio/wav"</td>
+  </tr>
+  <tr>
+    <td>`language`</td>
+    <td>Optional</td>
+    <td>The ISO 639-3 code for the language of the response, if known.</td>
+    <td>{'language':'eng'}</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### geo_point
+
+Represents a geospatial coordinate on the surface of the earth.
+
+#### Response format
+
+An array of two floating point numbers with the latitude and longitude: `[lat, long]`
+
+```
+[52.0835780,-106.6104880]
+```
+
+#### Type options (type_options)
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+#### Response Metadata
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`address`</td>
+    <td>Optional</td>
+    <td>TODO: is this useful?</td>
+    <td>"Plot 41, Kotei Residential Rd, Kotei, Kumasi, Ashanti Region, Ghana"</td>
+  </tr>
+  <tr>
```

Comment:
My take would be that in order to keep the spec focussed we leave this out of scope... since there could be many types of presentation encoding options that we could get drawn into.  It also might be a good principle to separate data and views.